lib/fs.dart

```diff
@@ -73,19 +73,20 @@ class FS {
   static Future<void> networkEvent({
     required String url,
     required String method,
-    int statusCode = 0,
-    int durationMs = 0,
-    int requestSize = 0,
-    int responseSize = 0,
+    int? statusCode = 0,
+    int? durationMs = 0,
+    int? requestSize = 0,
+    int? responseSize = 0,
```

Comment:
Does the `= 0` only apply when the param is omitted entirely, not when it's a `null` value?

Reply:
Yeah, correct, which is a little annoying. I have it set up here so it doesn't matter, probably could remove these default values since they're no longer very useful.